### DISCUSSION TITLE
help!!

### DISCUSSION THREAD

**Discussion by 95CBE** [Chia-Network/chia-blockchain#4859] [Support]
После изменения конфигурации системы появляется при запуске кошелька ERROR 13. Что делать,не могу указать директорию с плотами.

**Comment by SebastianPereiraa** [comment]
Попробуй в конфиге удалить диски на которых плотишь , потом их можно добавит ... 
по-моему должно помочь

**Reply by elmans1961** [reply]
Не скажите почему у меня в лог файле нет harveter  со все его процессом,а есть только timelord c процесами как пример из лога:5-14T00:32:54.504 timelord chia.timelord.timelord   : WARNING  Will not infuse a5f188588eюююююcdb  because its reward chain challenge is not in the chain
У меня работает harveter или нет?
Спасибо

**Reply by SebastianPereiraa** [reply]
В дебаге надо варнинг на инфо поменять

**Reply by elmans1961** [reply]
Это ничего не даст.Смена слов с WARNING(предостережение,предупреждение) на INFO(информация).И потом вы этим замените только верхнию часть лога,а потом она будет писать тот же WARNING,та как это зпложено я думаю в config.yaml
А еще какие мысли.Я смотрю у меня в основном эти ворнинги.Хавестеров с гулькин нос и то говорит,что надо уменьшить до менее 5 секунд обработку плотов,так как плот может не получить reward(вознаграждение).

**Reply by SebastianPereiraa** [reply]
Я не пойму что вы хотите, если посмотреть отклик плотов , то нужно заменить слова ...Затем в повершеле провисать нужные команды .

**Reply by 95CBE** [reply]
> Попробуй в конфиге удалить диски на которых плотишь , потом их можно добавит ...
> Зураб М.В.

спасибо, удалил папку .chia , перезапустил программу, все работает без ошибки,плоты прописались :)

**Reply by elmans1961** [reply]
"Затем в повершеле провисать нужные команды ."Скажите,пожалуйста,а какие команды прописывать в cmd или в powershell?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
